### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by luomai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +18,11 @@ Hence, we develop OpenPose-Plus, a fast and flexible pose estimation framework t
...
-
 This project is still under active development, some of the TODOs are as follows:
 - Parallel training (experimental)
 - Pose Proposal Networks, ECCV 2018
 
-## Training your model
+## Custom Model Training
 
 Training the model is implemented using TensorFlow. To run `train.py`, you would need to install packages, shown
 in [requirements.txt](https://github.com/tensorlayer/openpose-plus/blob/master/requirements.txt), in your virtual environment (Python <=3.6):
```
Comment (zsdonghao) [inline README.md]: could us install using : pip install requirements.txt    ?

File: README.md [hunk 2]
```diff
@@ -57,7 +53,7 @@ Train your model by simply running:
 python train.py
 ```
 
-## Parallel training
+## Training using Multiple GPUs
 
 We use Horovod to support train multiple tensorflow programs.
```
Comment (zsdonghao) [inline README.md]: TensorFlow

File: README.md [hunk 3]
```diff
@@ -92,7 +88,7 @@ $ mpirun -np 16 \
 ```
 
 
-## Inference
+## Inference on TensorRT
 
 Currently we provide two C++ APIs for inference, both defined in `include/openpose-plus.hpp`.
 They are for running the tensorflow model with tensorRT and post-processing respectively.
```
Comment (zsdonghao) [inline README.md]: TensorFlow , TensorRT

File: README.md (hunk 1 above)
Comment (luomai) [reply README.md]: already show this

**Pull request merged**